api/src/main/java/jakarta/data/repository/Repository.java

```diff
@@ -317,6 +317,23 @@
  *
  * </table>
  *
+ * <h3>Reserved for Future Use</h3>
```

Comment:
Should we have something for the future?

I mean, how can we make sure about this?

I feel more comfortable with the current code support.

Reply:
> Should we have something for the future?

Given that it gives the potential benefit for forward compatibility with future releases without any real downside, yes.  Various specifications that have keywords do this sort of thing.  Java even has some currently unused keywords.  I think the better question is whether more keywords should be included in our list.

> I mean, how can we make sure about this?

What are you unsure about? It doesn't burden providers at all. They don't need to do anything.  If we ever want to add one of these in a future release, we'll be able to do it in a point release and won't be forced into a major release just to add it, and we won't need to worry about breaking applications.  If we don't ever want to any of them, we don't need to.

Reply:
Otavio indicated he is okay with merging this one on today's Jakarta Data call, so I'm resolving this conversation and merging it.